modules/core.py

```diff
@@ -388,8 +389,8 @@ async def on_member_update(self, before, after):
                 after_name = discord.utils.escape_markdown(after.nick)
 
             embed = discord.Embed(color=0x9535EC, timestamp=datetime.datetime.utcnow())
-            embed.set_author(name=f'{str(before)} ({before.id})', icon_url=before.avatar_url)
-            embed.add_field(name='Before', value=before_name if not before.nick else before.nick, inline=False)
```

Comment:
Nickname needs to be escaped

Reply:
This is the before, I cleaned it because it was redundant consider the check above that escapes the nickname
```py
if not before.nick:
    before_name = before.name
else:
    before_name = discord.utils.escape_markdown(before.nick)